EXERCISE-3.md

```diff
@@ -69,7 +69,7 @@ In the following, you will be asked to apply a few simple changes to the
 `PipelineInterpreter` implementation:
 
 1. Open file:
-   `app/src/main/java/org/onosproject/ngsdn/tutorial/pipeconf/InterpreterImpl.java`
+   `app/src/main/java/org.onosproject.ngsdn.tutorial.pipeconf/InterpreterImpl.java`
```

Comment:
This is wrong, the actual path in the file system uses slashes. Dot notation is used for the Java package name.